### PR TITLE
* #95: Added CURRENT_CLUSTER function support.

### LINKED ISSUE #95: Add new function CURRENT_CLUSTER
See: https://github.com/exasol/virtual-schema-common-java/issues/236

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #95

### CODE REVIEW

File: doc/changes/changes_1.0.2.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Exasol Row Level Security (Lua) 1.0.2, released 2021-??-??
```
Comment (redcatbear) [inline doc/changes/changes_1.0.2.md:1]:
```suggestion
# Exasol Row Level Security (Lua) 1.0.2, released 2021-11-??
```

File: doc/changes/changes_1.0.2.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Code name: 
+
+## Summary
+
+## Features
+
+* #95: Added CURRENT_CLUSTER function support.
```
Comment (redcatbear) [inline doc/changes/changes_1.0.2.md:9]:
```suggestion
* #95: Added `CURRENT_CLUSTER` function support.
```

File: doc/developer_guide/developer_guide.md [hunk 3]
```diff
@@ -38,6 +38,7 @@ sudo luarocks install remotelog
 sudo luarocks install luacov
 sudo luarocks install luacov-coveralls
 sudo luarocks install luacheck
+sudo luarocks install exaerror
```
Comment (redcatbear) [inline doc/developer_guide/developer_guide.md:41]: Good point.

File: pom.xml [hunk 4]
```diff
@@ -5,15 +5,15 @@
...
     <artifactId>row-level-security-lua</artifactId>
-    <version>1.0.1</version>
+    <version>1.0.2</version>
     <name>Exasol Row Level Security (Lua)</name>
     <description>This projects provides a plug-in to the Exasol database that adds per-row access control.</description>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
-        <junit.version>5.7.2</junit.version>
```
Comment (redcatbear) [inline pom.xml:15]: Does Surefire find the integration test with 5.8.1? Never worked for me so far. Please make sure.

File: src/main/lua/exasolvs/query_renderer.lua [hunk 5]
```diff
@@ -57,15 +57,15 @@ end
 -- @return new query renderer instance
 --
 function M.new (query)
-    local self = {original_query = query, query_elements = {}}
+    local self = { original_query = query, query_elements = {} }
```
Comment (redcatbear) [inline src/main/lua/exasolvs/query_renderer.lua:60]:
```suggestion
    local self = {original_query = query, query_elements = {}}
```

Please keep the style uniform.

File: src/main/lua/exasolvs/query_renderer.lua [hunk 6]
```diff
@@ -57,15 +57,15 @@ end
...
     local OPERATORS = {
         predicate_equal = "=", predicate_notequal = "<>", predicate_less = "<", predicate_greater = ">",
         predicate_and = "AND", predicate_or = "OR", predicate_not = "NOT"
     }
 
     -- forward declarations
     local append_unary_predicate, append_binary_predicate, append_iterated_predicate, append_expression,
-        append_predicate_in, append_select, append_sub_select
+    append_predicate_in, append_select, append_sub_select
```
Comment (redcatbear) [inline src/main/lua/exasolvs/query_renderer.lua:68]:
```suggestion
        append_predicate_in, append_select, append_sub_select
```

Comment (redcatbear) [inline src/main/lua/exasolvs/query_renderer.lua:68]: Wrapped lines should get extra indentation. Is that IntelliJ changing the format?

File: src/main/lua/exasolvs/query_renderer.lua [hunk 7]
```diff
@@ -487,7 +487,7 @@ function M.new (query)
         return table.concat(self.query_elements, "")
     end
 
-    return {render = render}
+    return { render = render }
```
Comment (redcatbear) [inline src/main/lua/exasolvs/query_renderer.lua:490]:
```suggestion
    return {render = render}
```

Comment (redcatbear) [reply src/main/lua/exasolvs/query_renderer.lua:490]: Please check if this is configurable in your formatter.

File: src/test/lua/exasolvs/test_query_renderer.lua [hunk 8]
```diff
@@ -1389,6 +1389,15 @@ function test_query_renderer.test_current_user()
 
 end
 
+function test_query_renderer.test_current_cluster()
+    local original_query = {
+        type = "select",
+        selectList = {{type = "function_scalar", name = "CURRENT_CLUSTER"}}
+    }
+    assert_renders_to(original_query, 'SELECT CURRENT_CLUSTER')
+
```
Comment (redcatbear) [inline src/test/lua/exasolvs/test_query_renderer.lua:1398]: Remove empty line.

**Review by redcatbear — CHANGES_REQUESTED** [review]
Only a few formatting changes are required.

File: pom.xml (hunk 4 above)
Comment (AnastasiiaSergienko) [reply pom.xml:15]: Integration tests were running on the last commit

Comment (redcatbear) [reply pom.xml:15]: :+1:

**Review by redcatbear — APPROVED** [review]

**Pull request merged**